omim2obo/main.py

```diff
@@ -280,49 +280,61 @@ def omim2obo(use_cache: bool = False):
             graph.add((OMIM[mim_number], RDFS.subClassOf, OMIMPS[ps_id]))
 
     # Morbid map
-    morbid_map: Dict = parse_morbid_map(get_mim_file('morbidmap', download_files_tf))
-    for mim_number, mim_data in morbid_map.items():
-        # todo?: unused `mim_data` keys. Should they be used?
-        #  - phenotype_label: Similar to p_lab in 'assocs', but has more info
-        #  - gene_symbols
-        # - add cyto location
-        cyto_location: str = mim_data['cyto_location']
-        if cyto_location:
-            # What's 9606chr - joeflack4 2021/11/11
-            chr_id = '9606chr' + cyto_location
-            # RO:0002525 (is subsequence of)
-            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
-            add_subclassof_restriction(graph, RO['0002525'], CHR[chr_id], OMIM[mim_number])
-        assocs: List[Dict] = mim_data['phenotype_associations']
-        # - add associations
+    gene_phenotypes: Dict[str, Dict] = parse_morbid_map(get_mim_file('morbidmap', download_files_tf))
+
+    # - Cyto location: Add RO:0002525 (is subsequence of)
+    # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
+    for gene_mim, gene_data in gene_phenotypes.items():
+        if gene_data['cyto_location']:
+            chr_id = '9606chr' + gene_data['cyto_location']  # todo: document meaning of 9606chr
+            add_subclassof_restriction(graph, RO['0002525'], CHR[chr_id], OMIM[gene_mim])
+
+    # TODO: @Trish: I've removed addition of gene->disease relations, leaving only disease->gene. Is that what we want?
+    #  Removed because extra complexity. Want to get down disease->gene code first. And Mondo doesn't need them.
```

Comment:
We discussed calculating the number of associations [here](https://github.com/monarch-initiative/omim/pull/158/files#diff-6a2c7728d655af0d58b05a0d99cc2865c94c355a91d8df1d647a002f7eee6a4eL307) differently since it did not appear that if a gene was unique in the morbidmap file that it was sufficient to tag that gene as causal for a disease, and instead that the total number of genes per phenotype mim number should be used to determine if a gene is causal (ie only 1 gene per disease).

Reply:
1. Gene removal
There are a few reasons. It's been a year since this was written, so I don't remember everything, but I keep being reminded about the disease-centric nature of things, and I wonder whether we meant to add them in the first place. And moreso, I want this code review to be easier at first; for us to get the disease-gene stuff handled first. If you look at the code, I had to add on a bit to what you wrote to complete it.
Thinking more about it today though, I did leave part of the gene restrictions; the "cyto location" bit. And for adding the other ones back, I think the best way to handle it is to add some extra lines to this "cyto location" block. It'll actually look very similar to what was there before; I just need to drop the "n associations" conditional for the genes as that only matters for the diseases.
TLDR: Let's make sure the Disease2Gene stuff is correct first and I can add back the Gene2Disease shortly.

2. Yes, AFAIK only `mondo-ingest` consumes these outputs.

Reply:
Added the gene disease relationships back. 

I had to put some thought into it to make sure I felt it was correct, and to make the code flow as cleanly and understandably as I could make it. 

Feel free to review here or during meeting.